convert_to_multi_instance.sh

```diff
@@ -817,6 +818,36 @@ function cleanup_webhook() {
 
 }
 
+function update_opreqs(){
+    title "Updating Operand Requests to use Operand Registry in new CS namespace"
+    #check map to namespaces, get list of requested from ns from there
+    #update opreq in map to first
+    #update opreq in list of namespace from first line
+    for csns in $map_to_cs_ns
+    do
+        local namespaces=$(${OC} get cm common-service-maps -o yaml -n kube-public | $YQ '.data[]' | $YQ '.namespaceMapping[] | select(.map-to-common-service-namespace == "'$csns'").requested-from-namespace' | awk '{print $2}' | tr '\n' ' ')
+        namespaces="$namespaces $csns"
+        for ns in $namespaces
+        do
```

Comment:
The loops are iterating over all namespaces in cs-maps. Why not avoid the loops entirely, and just update all OperandRequests to have `ibm-common-services` registryNamespace. It would be equivalent and faster

Reply:
The map_to_cs_ns variable only includes the map-to values that the topology check noted were changed so it is actually only the subset of namespaces used in this run of the conversion script. See line 664 and 665